server/src/main/java/org/tctalent/server/service/db/impl/PartnerServiceImpl.java

```diff
@@ -212,6 +212,17 @@ public Page<PartnerImpl> searchPaged(SearchPartnerRequest request) {
         partner.setAutoAssignable(request.isAutoAssignable());
         partner.setSourceCountries(sourceCountries);
 
+        if (request.getRedirectPartnerId() != null) {
+            PartnerImpl newPartner = manageRedirectPartnerAssignment(request.getRedirectPartnerId());
+            partner.setRedirectPartner(newPartner);
+
+            LogBuilder.builder(log)
+                .action("redirectInactivePartnerUrl")
+                .message("URLs identifying inactive partner " + partner.getName() +
+                    " will now redirect to " + newPartner.getName() + ".")
+                .logInfo();
+        } else partner.setRedirectPartner(null);
```

Comment:
Syntactically correct but usually not recommended.

Better to use braces for else.

See https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used

Reply:
```suggestion
        } else {
          partner.setRedirectPartner(null);
        }
```

Reply:
Thanks, @camerojo, yea I can see how it would be less readable! Request change committed. 👍